src/main/java/com/bringframework/registry/BeanDefinitionImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.bringframework.registry;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+import lombok.Builder;
+
+/**
+ * Implementation of {@link BeanDefinition}
+ * Represents the definition of a bean that is managed by the Bring IoC container.
+ */
+@Builder
+public class BeanDefinitionImpl implements BeanDefinition {
```

Comment:
@vitshch  do re really need BeanDefinition interface ?

Reply:
on daily meeting we decided to left this interface